tlsr9/ble/vendor/controller/b9x_bt_buffer.c

```diff
@@ -18,17 +18,23 @@
 
 #include "b9x_bt_buffer.h"
 
+#if defined(CONFIG_BOARD_TLSR9518ADK80D_RETENTION) || defined(CONFIG_BOARD_TLSR9528A_RETENTION)
+#define _attribute_ble_buff_sec_ __attribute__((section(".ram_dlm")))
+#else
+#define _attribute_ble_buff_sec_
+#endif /* CONFIG_BOARD_TLSR9518ADK80D_RETENTION || CONFIG_BOARD_TLSR9528A_RETENTION */
+
 /******* ACL connection LinkLayer TX & RX data FIFO allocation, Begin ********/
-u8 app_acl_rxfifo[ACL_RX_FIFO_SIZE * ACL_RX_FIFO_NUM] = {0};
+_attribute_ble_buff_sec_ u8 app_acl_rxfifo[ACL_RX_FIFO_SIZE * ACL_RX_FIFO_NUM] = {0};

```

Comment:
It seems here we can use .retention_data for all boards to avoid introducing new section.